docs/README.md

```diff
@@ -28,6 +28,7 @@ Here you can find the complete list of decorators, however we encourage you to r
   - [`@injectProps`](functions/injectProps.md)
   - [`@injectState`](functions/injectState.md)
   - [`@debounce(wait: number = 300, immediate: boolean = false)`](functions/debounce.md)
+	- [`@lock`](functions/lock.md)
```

Comment:
it seems it is placed in a sublist of `debounce` if you try to open the markdown preview